update-and-show-snapshots.js

```diff
@@ -0,0 +1,24 @@
+let { exec } = require('child_process')
+let { promisify } = require('util')
+let chalk = require('chalk')
+
+let showSnapshots = require('./show-snapshots')
+
+let execCommand = promisify(exec)
+
+async function updateAndShowSnaphots (print, error, cwd, filter) {
+  print(chalk.blue('\nUpdating snapshots... \n'))
+
+  let { stdout, stderr } = await execCommand('"./node_modules/.bin/jest" -u', {
```

Comment:
`npx jest --no-install` is a better way since we can’t trust to find `jest` in `./node_modules/.bin/`.

For instance, we `print-snapshots` can be run from `./test` dir. In this case `./node_modules/.bin/jest` will not be found.